ScienceJournal/Resources/Strings.bundle/en.lproj/Localizable.strings

```diff
@@ -1195,6 +1195,9 @@
 /* Long description of an unknown bluetooth sensor displayed below a representative image on the Learn More page [CHAR_LIMIT=NONE] */
 "sensor_desc_second_paragraph_unknown_bluetooth" = "Bluetooth is a wireless technology standard using radio waves to exchange data over short distances from fixed and mobile devices.";
 
+/* Message to display to the user when bluetooth is not on but a bluetooth sensor is in the experiment on Observe [CHAR_LIMIT=200] */
```

Comment:
Just for a bit of clarity, how about:

`/* Message to display to the user when Bluetooth is disabled but a Bluetooth sensor is listed in sensor settings because it has been connected in the past.`